sda-commons-asyncapi/README.md

```diff
@@ -0,0 +1,143 @@
+# SDA Commons Shared AsyncAPI
```

Comment:
Oh, more than I expected. I thought about using [SpringWolf](https://github.com/springwolf/springwolf-core) for AsyncAPI documentation. I got this as another missing feature for Spring Boot Commons in the back of my head. Let's discuss this in the team.

Reply:
I would agree that the AsyncAPI generation is beyond the scope of the current task. It also bloats this PR quite a lot. However, it is another dependency that is currently taken from our Dropwizard library and mixes up the dependency tree in the Spring services.

I heard mixed opinions about Spring Wolf from customers in our microservice context. In general the result seem to be _good enough_ but not completely satisfying. The AsyncAPIs we can produce with our current implementation are describing the API very well. There are still some edge cases that can't be documented as good as it should be (e.g. describing object type fields separately from the object definition). Also the library is quite old and should only be used in test code.

I think, it's not too bad to have a solution in place right away. But I would prefer further research to find a better (and actively maintained) solution. That may be Spring Wolf if we find a way to create a satisfying result that is easy to use. What do you think about marking the AsyncAPI module as a workaround alpha solution that is subject to be replaced without further notice?

Reply:
Yaml module removed.

Updated Async Readme

> ## ⚠️ Experimental ⚠️
>
> ### Please be aware that SDA SE is likely to change or remove this artifact in the future